### ISSUE TITLE
Special rule for Hungarian fixed holidays on Tuesdays and Thursdays

### ISSUE THREAD

**Issue by urbalazs** [opening-hours/opening_hours.js#449]
In Hungary, we have a special rule for public holidays: if a public holiday is on Tuesday or on Thursday, then Monday or Friday will be also public holiday (Hungarians call it "resting day", do not need to work), but a Saturday before or after will be working day and act as Monday or Friday.

Here is an example:

20th of August is a fixed date public holiday in Hungary. In 2024, this is on Tuesday. Because of this, 19th of August (Monday) is a "resting day" (basically a public holiday), but 3rd of August (Saturday) is a working day and everything is open as it would be a Monday.

In this year (2023) it was not an issue since 20th of August was on Sunday, which is already a not working day.

TL;DR: in 2024

- 3rd of August, Saturday is working day as it would be a Monday
- 19th of August, Monday is public holiday as it was already worked on 3rd of August
- 20th of August, Tuesday is a fixed date public holiday

Is there any chance to map such a variable date holiday and working day in this project?

**Comment by ypid** [comment]
Please first check if this is already implemented elsewhere, see #300
https://github.com/KDE/kholidays/blob/master/holidays/plan2/holiday_hu_hu

If you can express it in JavaScript, feel free to implement it. See function `getMovableEventsForYear` in https://github.com/opening-hours/opening_hours.js/blob/master/src/index.js.
